sites/main-site/src/content/blog/2024/seqera-containers-part-1.mdx

```diff
@@ -0,0 +1,360 @@
+
+Migrating nf-core to use Seqera Containers will make provisioning multi-package containers trivial. The module developer will only need to edit the module `environment.yml` file and everything else will be fully automated and available almost immediately. Packages can be used minutes after their release on Bioconda and we will even have automation in place to bump the Conda + Seqera Containers packages when a tool is updated.
+
+### BioContainers API
+
+To simplify adding single-package BioContainer images to nf-core modules, we have developed the nf-core/tools package to query the BioContainer API for a given tool version and fetch the image URIs. This process is repeated during linting of nf-core modules and pipelines, to ensure that there is not an accidental mismatch between Conda and Docker / Singularity software versions. Unfortunately, the BioContainer API has had a history of being slow and unreliable, with a lot of down time (204 error message anyone?). This has the knock-on effect of causing many CI tests to fail, which is frustrating and delays the process of pull-request merges.
+
+By migrating to Seqera Containers we will have similar automation when linting modules, but it will use the Seqera Wave API instead. This has been built to scale to very high volumes and has an extensive and robust back-end with a high degree of monitoring. Note that the Wave API will _not_ be used when people run nf-core pipelines, this is only developers running linting and GitHub Actions automation during development.
+
+### Reliability of hosting
+
+BioContainer Docker images are hosted publicly on [quay.io](http://quay.io). This service is provided free of charge, however in recent years we have had some reliability issues. This becomes more noticeable as the community grows and the number of users trying to pull images increases. Because quay.io is a huge service for whom we are only a tiny player, we have no recourse when this happens and have to just wait until it becomes available again.
+
+Seqera serves nf-core as its primary community: as such it should be in a position to respond immediately to any problems or needs. All Seqera Container images (Docker and Singularity) are hosted on custom infrastructure built by Seqera with hosting provided by AWS, so the whole stack is within reach.
+
+## Potential concerns
+
+<Image class="d-block ms-3 mb-3 float-end rounded w-25 shadow" src={chillies} alt="Photo by Anne Nygård on Unsplash." />
+
+### Will these be reproducible?
+
+Because the images are hosted on long-term infrastructure and their URLs are hardcoded into pipeline configuration, they should be highly reproducible. Even if new builds of the same software are created in the future using Wave (for example due to an update in the base infrastructure used by Wave) then the old images are still pinned by the pipeline and will continue to be used, much like the BioContainer URIs used currently.
```

Comment:
```suggestion
Seqera container images are hosted on long-term infrastructure and because their URLs will be hardcoded into pipeline configuration, they should be highly reproducible. Even if new builds of the same software are created in the future using Wave (for example due to an update in the base infrastructure used by Wave) then the old images are still pinned by the pipeline and will continue to be used, much like the BioContainer URIs used currently.
```